client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -118,6 +120,26 @@ class EditContactInfoFormCard extends React.Component {
 		}
 	}
 
+	hasEmailChanged() {
+		return this.props.contactInformation.email === formState.getFieldValue( this.state.form, 'email' );
+	}
```

Comment:
I'd prefer something like `isEmailSame`, `hasXXXChanged` sounds like this function is tracking a pristine/dirty state for the form whereas it's just an equality check.